plugins/helphub-post-types/classes/class-helphub-post-types-post-type.php

```diff
@@ -406,29 +417,21 @@ public function meta_box_content_render( $field_data ) {
 						}
 						$html .= '</td></tr>' . "\n";
 						break;
-					case 'select':
-						$field = '<select name="' . esc_attr( $k ) . '" id="' . esc_attr( $k ) . '" >'. "\n";
+					case 'select' :
```

Comment:
Case statements should not have a space before the colon, i.e.` 'case 'select':`